### DISCUSSION TITLE
Is there any way to block client with certain IPs?

### DISCUSSION THREAD

**Discussion by squromiv** [TechnitiumSoftware/DnsServer#808] [Q&A] [answered]
I have noticed, that there are thousands of queries to my Technitium DNS instance port 53 from third-party clients. Is it possible to block them in Technitium in order not to adjust firewall rules?

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. Yes, you can do that by using the Drop Requests DNS App and configure it to block the network addresses or IP addresses you wish to block. The app will simple drop the requests and it wont reflect in Dashboard stats.
